Backpack-SwiftUI/MapMarker/Classes/BPKPriceMapMarker.swift

```diff
@@ -0,0 +1,120 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKPriceMapMarker: View {
+    public enum State {
+        case `default`
+        case focused
+        case viewed
+        case disabled
+        
+        var fontStyle: BPKFontStyle {
+            switch self {
+            case .default:
+                return .label2
+            case .focused:
+                return .label1
+            case .viewed:
+                return .label2
+            case .disabled:
+                return .label2
```

Comment:
Can we change this to:
```suggestion
            case .focused:
                return .label1
            default:
                return .label2
```